amt/schema/system_card.py

```diff
@@ -19,4 +21,6 @@ class SystemCard(BaseModel):
     labels: list[dict[str, Any]] = Field(default=[])
     status: str = Field(default=None)
     instruments: list[InstrumentBase] = Field(default=[])
+    requirements: list[RequirementTask] = Field(default=[])
+    measures: list[MeasureTask] = Field(default=[])
```

Comment:
did we update the schema as well?

Reply:
Yes :) Hence the newly introduced RequirementTask and MeasureTask with a RequirementBase and Requirement and MeasureBase and Measure